TKeazirian.HTTPServer/Constants.cs

```diff
@@ -0,0 +1,11 @@
+namespace TKeazirian.HTTPServer
+{
+    public static class Constants
+    {
+        public const string NewLine = "\r\n";
+        public const string BodySeparator = "\r\n\r\n";
+        public const string Space = " ";
+        public const string Status200 = "HTTP/1.1 200 OK";
+        public const string Status404 = "HTTP/1.1 404 Not Found";
```

Comment:
For the statuses have you considered an [enum](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/enum)?